climate_emotions_map/make_map.py

```diff
@@ -254,7 +270,10 @@ def make_map(
     fig.add_scattergeo(
         locations=state_abbrevs_long["state_abbreviated"],
         locationmode="USA-states",
-        text=state_abbrevs_long["state_abbreviated"],
+        text=[
+            f"<b>{abbr}</b>"
+            for abbr in state_abbrevs_long["state_abbreviated"]
+        ],
```

Comment:
slight efficiency improvement
```suggestion
        text=state_abbrevs_long["state_abbreviated"].apply(lambda abbr: f"<b>{abbr}</b>"),
```